sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 1.1.0-beta.1 (Unreleased)
```

Comment:
It looks like the changelog wasn't updated before 1.0.5 was released.

Reply:
Yeah, this was missed on the release branch, I will update it now separately.